packages/apps/spaces/app/organizations/src/components/Columns/Cells/organization/OrganizationCell.tsx

```diff
@@ -65,7 +67,7 @@ export const OrganizationCell = ({
             {isHovered && (
               <Flex
                 position='absolute'
-                bottom='16px'
+                bottom='8px'
```

Comment:
px value in this case should be changed to `2` (prefer theme value over magic numbers whenever possible)